collect_app/src/main/java/org/odk/collect/android/map/MapPoint.java

```diff
@@ -0,0 +1,58 @@
+package org.odk.collect.android.map;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import java.util.Locale;
+
+public class MapPoint implements Parcelable {
+    public final double lat;
+    public final double lon;
+
+    public MapPoint(double lat, double lon) {
+        this.lat = lat;
+        this.lon = lon;
+    }
+
+    public MapPoint(Parcel parcel) {
```

Comment:
I think this constructor should be private so that only the `CREATOR` field can access. Am I right?

Reply:
Yes, good idea!  Fixed.